guide/general/gear.md

```diff
@@ -314,7 +318,7 @@ Item | Stats | Location
 
 Item | Stats | Location
 :---: | :---: | :---:
-{{ site.data.gear.erethos_the_empty_promise }} | Vers (67%) > Hast | Scalecommander Sarkareth
+{{ site.data.gear.erethos_the_empty_promise }} | Hast (67%) > Mast | Scalecommander Sarkareth
```

Comment:
Is there any specific reason to not write the full stat name?

Reply:
Yeah I just wanted the 4 stats to be 4 letters wide. 
No real reason appart from that so I can switch it back if you want.